### PR TITLE
2860 account switch tooltip should not hide user menu

### LINKED ISSUE #2860: [react-cat][1.5] Account switch tooltip should not hide user menu
**Describe the bug**
Switching to a different account shows a toolip, which hides the user menu entry

**To Reproduce**
1. Click on username to switch account
2. Click on username to switch account again

**Expected behavior**
Move tooltip to the left to not hide user menu

**Screenshots**
acc-switch-tooltip

### PR DESCRIPTION
**Pull request by react-cat**
<h2>General</h2>
Closes #2860

Add auto close of account notification when user open menu/account management dropdown or open account unlock modal

<h2>General</h2>
Please make sure the following is done:

- Pull request is onto develop
- If you haven't already, have a look at
  - https://github.com/bitshares/bitshares-ui/blob/develop/CODE_OF_CONDUCT.md
  - https://github.com/bitshares/bitshares-ui/blob/develop/CONTRIBUTING.md

<h2>Code Preparation</h2>

_Please review all your changes one last time before committing_

- [x] Check for unused code
- [x] No unrelated changes are included
- [x] None of the changed files are reformatting only
- [x] Code is self explanatory or documented
- [x] All written text is properly translated (english language)

<h2>Testing</h2>

_The branch has been tested on the following browsers (desktop and mobile view)_

- [x] Chrome 
- [x] Opera
- [x] Firefox
- [x] Safari

Before: 
AwesomeScreenshot-2019-7-22-1563803691410

After:
AwesomeScreenshot-2019-7-22-1563803429697

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ echo TRAVIS_TAG=$TRAVIS_TAG
 echo TRAVIS_BRANCH=$TRAVIS_BRANCH
 echo TRAVIS_PULL_REQUEST_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
 
-if [[ "$unamestr" == 'Linux' && -n $TRAVIS_TAG && $TRAVIS_BRANCH =~ [2-9]\.[0-9]\.[0-9]{6}$ ]]
+if [[ "$unamestr" == 'Linux' && -n $TRAVIS_TAG && $TRAVIS_BRANCH =~ [2-9]\.[0-9]\.[0-9]{6}\.?[0-9]? ]]
```
Comment (startailcoon) [inline deploy.sh]: What is this change for, accidental push?

Comment (react-cat) [reply deploy.sh]: @startailcoon yep, sorry

Comment (startailcoon) [reply deploy.sh]: Please commit a change to remove undesired changes.

Comment (react-cat) [reply deploy.sh]: @startailcoon Done

File: app/components/Layout/Header.jsx [hunk 2]
```diff
@@ -52,7 +52,8 @@ class Header extends React.Component {
             isDepositModalVisible: false,
             hasDepositModalBeenShown: false,
             isWithdrawModalVisible: false,
-            hasWithdrawalModalBeenShown: false
+            hasWithdrawalModalBeenShown: false,
+            accountNotificationActiveKeys: []
```
Comment (sschiessl-bcp) [inline app/components/Layout/Header.jsx]: This state is not used for rendering at all and will cause unnecessary re-renders. Please move into a normal object field.

Comment (react-cat) [reply app/components/Layout/Header.jsx]: @sschiessl-bcp Done

**Pull request merged**